initial_pose:
  position:
    x: 0.30699088606283903
    y: 2.8203182887441273e-05
    z: 0.4870162491576909
  orientation:
    x: 1.0
    y: 0.0
    z: 0.0
    w: 0.0
